### DISCUSSION TITLE
But it didn't start for me(

### DISCUSSION THREAD

**Discussion by HughVR** [fholger/crysis_vrmod#8] [General]
Thanks for the mod! But it didn't start for me(. I do everything clearly according to the instructions with the game that you specified: on the monitor, the game starts with the VR mod logo, the sound on the Oqulus 2 goes, but steam VR makes attempts to launch in the helmet, for a couple of seconds the same rectangle of waiting for Crysis to launch is shown, then the endless plain of steam VR. What could be the problem?

**Comment by polybasss** [comment]
the same thing happens to me with Quest 2 and air link

**Reply by Davros19** [reply]
not sure if it is the same issue but try adding -dx9 to the target in the shortcut

**Comment by fholger** [comment]
Could you try the latest development build and let me know if that helps? Download this one: https://github.com/fholger/crysis_vrmod/releases/download/latest/crysis-vrmod-0.1.1.exe

**Comment by HughVR** [comment]
I tried the new version of the mod. Nothing has changed(

**Comment by polybasss** [comment]
tried the latest version and everything remains the same, I have a partner with quest2 also that works for him, GOG version in case it helps and appears in mods

**Comment by HughVR** [comment]
Yes, and I have the GOG version

**Comment by fholger** [comment]
Could you post your Game.cfg files (located in `My Documents\My Games\Crysis\game.cfg`)?

**Comment by fholger** [comment]
You could also try the updated dev build again (same link as above); I've had someone report it's working for him now, even though I don't really understand why.

**Reply by polybasss** [reply]
> También puede probar la compilación de desarrollo actualizada nuevamente (el mismo enlace que arriba); Alguien me informó que ahora le está funcionando, aunque realmente no entiendo por qué.
still doesn't work for me...


-- [Game-Configuration]
-- Attention: This file is re-generated by the system! Editing is not recommended! 

cl_bob = 1
cl_controllersensitivity = 45
cl_invertcontroller = 0
cl_invertmouse = 0
cl_sensitivity = 45
cl_voice_volume = 1
dt_enable = 0
g_blood = 0
g_buddymessagesingame = 1
g_displayignorelist = 1
g_enablealternateironsight = 0
g_pstutorial_enabled = 1
g_quickgame_map = "multiplayer/ia/armada"
g_quickgame_min_players = 1
g_quickgame_mode = "InstantAction"
g_quickgame_prefer_favorites = 1
g_quickgame_prefer_lan = 1
g_quickgame_prefer_mycountry = 1
g_startfirsttime = 0
hud_crosshair = 6
hud_showallobjectives = 0
hud_subtitles = 0
i_forcefeedback = 1
i_mouse_smooth = 0
net_enable_voice_chat = 1
r_brightness = 0.5
r_contrast = 0.5
r_fullscreen = 1
r_gamma = 1
r_height = 768
r_motionblurshutterspeed = 0.015
r_vsync = 0
r_width = 1024
s_dialogvolume = 1
s_musicvolume = 0.6
s_sfxvolume = 1
sys_spec_gameeffects = 3
sys_spec_motionblur = 4
sys_spec_objectdetail = 3
sys_spec_particles = 3
sys_spec_physics = 3
sys_spec_postprocessing = 3
sys_spec_shading = 3
sys_spec_shadows = 3
sys_spec_sound = 3
sys_spec_texture = 3
sys_spec_volumetriceffects = 3
sys_spec_water = 3

**Comment by fholger** [comment]
For completeness, please also post your Game.log from your Crysis install folder. Though unfortunately at this point I'm out of ideas.

**Reply by polybasss** [reply]
[Game Build(6156) Date(10 Apr 2023) Time(02 22 57).log](https://github.com/fholger/crysis_vrmod/files/11189587/Game.Build.6156.Date.10.Apr.2023.Time.02.22.57.log)

**Reply by polybasss** [reply]
I have tried with Hp reverb G2 and the same thing happens to me


[Game.log](https://github.com/fholger/crysis_vrmod/files/11189897/Game.log)

**Comment by fholger** [comment]
Could you please take a new log file with the new release (https://github.com/fholger/crysis_vrmod/releases/download/v0.1.2/crysis-vrmod-0.1.2.exe) and post it here? The new release is unlikely to work, either, but I need the new log to hopefully understand why it is crashing for you.

**Comment by HughVR** [comment]
The reason is found! Your mod conflicts with MSI Afterburner. If you turn it off, everything starts normally. Thanks for the great mod! A reason to go through a wonderful game again) Are you planning to make VR for the rest of the games in the series? It would also be great, of course, to remake Far Cry into a viar!!!

**Reply by polybasss** [reply]
Thank you very much for finding the solution, I have already tried it and it is working with both Quest 2 and Reverb G2

**Comment by fholger** [comment]
I might do Far Cry 1, in fact, I originally planned to do Far Cry instead of Crysis, but the Far Cry SDK situation is a bit of a mess. That's why I switched to Crysis :) The other parts of both series, however, never received an SDK, so unfortunately I cannot do anything for them.

**Reply by Braxusx** [reply]
I would love to see Far Cry! But I am so grateful you're tackling Crysis as well. It's a splendid game to play in VR. It's wonderful modders like you that really keep PCVR alive and well. Thank you so much!

**Comment by HughVR** [comment]
Are you going to tighten the control of VR trackers? If yes, then please make it possible to switch controls for left-handed people: movement with the right stick, and turns and weapons with the left!!! And in VR Crysis, aiming \ turning is very inconvenient. How to make mouse turns like the original game?

**Reply by DJAlexBlanco** [reply]
Interesting. I think you're the first left handed VR gamer I've encountered who wants to switch the thumbstick! I thought it was just something right handed people thought left handed people wanted! ;-) I'm a lefty, and I know three other lefty gamers, and none of us want to flip the sticks around. We've all been playing for so long with a regular gamepad that to switch the sticks in VR feels really weird!

**Comment by fholger** [comment]
See https://github.com/fholger/crysis_vrmod/issues/3

**Comment by xPawelx** [comment]
quest 2 - I had to do fresh install and than It works
